pypeit/sensfunc.py

```diff
@@ -170,6 +171,9 @@ def __init__(self, spec1dfile, sensfile, par=None, debug=False):
         self.par = self.spectrograph.default_pypeit_par()['sensfunc'] if par is None else par
         # TODO: Check the type of the parameter object?
 
+        # Set the algorithm in the datamodel
+        self.algorithm = self.__class__.algorithm
```

Comment:
I'm worried that naming the instance attribute the same as the class attribute might come back to bite us.  Can we instead rename the class attribute to `_algorithm` or something?

Reply:
yeah, sounds good